spec/prog/vm/update_ipv6_spec.rb

```diff
@@ -75,9 +65,9 @@ def create_load_balancer(cert_enabled:)
   end
 
   it "stops services and cleans up namespace" do
-    stub_vm_host_for_ssh
     create_load_balancer(cert_enabled: true)
     inhost_name = vm.inhost_name
+    sshable = pr.vm_host.sshable
```

Comment:
This appears to be the same in every spec. I would add an `sshable` method or `let(:sshable)` that returned `pr.vm_host.sshable`, and drop the lines in each spec.